books/free-programming-books-langs.md

```diff
@@ -1109,6 +1109,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [The JavaScript Way](https://github.com/bpesquet/thejsway) - Baptiste Pesquet
 * [The Problem with Native JavaScript APIs](https://www.oreilly.com/programming/free/native-javascript-apis.csp) - Nicholas C. Zakas (PDF, email address *requested*)
 * [Thinking in JavaScript](https://www.amazon.com/Thinking-JavaScript-Aravind-Shenoy-ebook/dp/B00JUI6LUQ) - Aravind Shenoy (Kindle)
+* [TypeScript Notes for Professionals](https://books.goalkicker.com/TypeScriptBook2/) - Compiled from StackOverflow Documentation (PDF)
```

Comment:
Why is this not in the typescript section?

Reply:
Was my mistake :worried: don't check the section after movement. Solving...